cmd/bot/run.go

```diff
@@ -112,7 +112,7 @@ func ProcessUnprovenBotDelegatedWithdrawals(ctx context.Context, log log.Logger,
 				// Since the unproven withdrawals are sorted by the on-chain order, we can break here because we know
 				// that the subsequent of the withdrawals are not ready to be proven yet.
 				return
-			} else if strings.Contains(err.Error(), "execution reverted") {
+			} else if strings.Contains(err.Error(), "execution reverted") || strings.Contains(err.Error(), "filtered") {
```

Comment:
Consider these scenarios
- If we want to check the failed cases manually, can we do it conveniently? In current design, we need to filter the `failure_reason` field by checking inclusion of substring `filtered`, which might be inefficient for database.
- If we checked the failed case and it's expected, can we mark it acknowledged conveniently?
- If the failure is not expected, can we retry it conveniently? It will be better if we can update the database and it will be retried automatically.

Plz reconsider the db schema design based on the scenarios above. Maybe we can add more fields in db to indicate the different status. Feel free to have more discussion.

Reply:
> `failure_reason` aims to record exceptional cases.
> 
> In my opinion, these exceptional scenario should rarely occur, so we don't have to optimize the query performance at this moment.

Sure, let's leave it for the future.